examples/mqtt/dcc_comms/iotcc/dell5k_edge_system_iotcc_mqtt.py

```diff
@@ -0,0 +1,192 @@
+# -*- coding: utf-8 -*-
```

Comment:
This is a user program so we are keeping them in the MQTT folder for better accessibility. Packages are placed together.